hal/src/main/native/systemcore/CAN.cpp

```diff
@@ -108,21 +109,24 @@ bool SocketCanState::InitializeBuses() {
     int32_t status = 0;
     HAL_SetCurrentThreadPriority(true, 50, &status);
     if (status != 0) {
-      std::printf("Failed to set CAN thread priority\n");
+      fmt::print("Failed to set CAN thread priority\n");
```

Comment:
```suggestion
      wpi::println("Failed to set CAN thread priority");
```

Reply:
wpi::println doesn't seem to exist anywhere. But I'll switch to wpi::print

Reply:
They're here: https://github.com/wpilibsuite/allwpilib/blob/2027/wpiutil/src/main/native/include/wpi/print.h#L41-L61

Reply:
Oh. But we literally use it nowhere. Wasn't fmt::println slow, which is why we never use it?

Reply:
Maybe, but it seems fine now. println dispatches to vprintln:
```cpp
FMT_FUNC void vprintln(std::FILE* f, string_view fmt, format_args args) {
  auto buffer = memory_buffer();
  detail::vformat_to(buffer, fmt, args);
  buffer.push_back('\n');
  detail::print(f, {buffer.data(), buffer.size()});
}
```